build.gradle

```diff
@@ -1,10 +1,10 @@
 plugins {
-    id 'java'
-    id 'groovy'
-    id 'maven'
-    id 'maven-publish'
-    id 'idea'
-    id 'eclipse'
+    id("java")
```

Comment:
This isn't how the Gradle DSL is intended to be used, and furthermore that style of method calling is not used throughout the buildscript.

As you mention on IRC, this could be used in preparation of migrating to the Kotlin DSL - however, we've yet to make a decision on whether we are moving to that, and regardless until we do so we should maintain the standards of what we're using.